### DISCUSSION TITLE
If `scoop uninstall scoop` fails, do the following steps completely uninstall Scoop?

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [ScoopInstaller/Scoop#6190] [Help and Questions]
If `scoop uninstall scoop` fails (see https://github.com/ScoopInstaller/Scoop/issues/6189), does simply deleting the following two folders (and their subfolders) completely uninstall Scoop, or is there more to it?

```
~/.config/scoop
~/scoop
```

**Comment by HUMORCE** [comment]
`C:\ProgramData\scoop`, if a global installation was performed before.
